glean-core/python/glean/glean.py

```diff
@@ -132,56 +132,60 @@ def initialize(
 
         cls._upload_enabled = upload_enabled
 
-        cfg = _ffi.make_config(
-            cls._data_dir,
-            application_id,
-            cls._upload_enabled,
-            configuration.max_events,
-        )
+        # Use `Glean._execute_task` rather than `Glean.launch` here, since we
+        # never want to put this work on the `Dispatcher._preinit_queue`.
+        @Dispatcher._execute_task
+        def initialize():
+            cfg = _ffi.make_config(
```

Comment:
I just noticed we're not calling `register_pings` as we do for other platforms. I believe that's ok, since we are not currently sending any ping, but this might change with [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1639882). Nothing to do here?

Reply:
There's no need for the register_pings static workaround in Python, since pings are always explicitly registered when they are loaded with `load_pings`.  Maybe worth a comment here, though.

Reply:
Yes, having a comment here would be really helpful!

Reply:
Ok.  Adding one in this PR.